gui/wxpython/psmap/frame.py

```diff
@@ -23,6 +23,7 @@
     import queue as Queue
 from math import sin, cos, pi, sqrt
 
+import numpy as np
```

Comment:
Since this is needed just in case of landscape, I would import it just before np is used instead of the global import.

Reply:
I have already fixed it.